### ISSUE TITLE
NTR: Rapid gastric emptying

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10617]
**Preferred term label:**
Rapid gastric emptying

**Synonyms**
Dumping syndrome

**Definition (free text, please give PubMed ID)**
PMID: 16207692
Dumping is the effect of alteration in the motor functions of the stomach, including disturbances in the gastric reservoir and transporting function. Anatomic and physiologic changes introduced by gastric surgery result in clinically significant dumping syndrome in approximately 10% of patients. Gastrointestinal hormones play an important role in dumping by mediating responses to surgical resection. Treatment options of dumping syndrome include diet, medications, and surgical revision

**Parent term (use hpo.jax.org/app)**
Functional abnormality of the gastrointestinal tract HP:0012719

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Adding new term: 
- ID: HP:6001017
- label: Rapid gastric emptying
- parents: [Abnormal gastrointestinal transit time (HP:0030896)](https://hpo.jax.org/app/browse/term/HP:0030896)
- pmid: PMID:16207692
- def: A condition in which food moves through the stomach into the small intestine at an abnormally fast rate.
- comment: Dumping is the effect of alteration in the motor functions of the stomach, including disturbances in the gastric reservoir and transporting function. Anatomic and physiologic changes introduced by gastric surgery result in clinically significant dumping syndrome in approximately 10% of patients. Gastrointestinal hormones play an important role in dumping by mediating responses to surgical resection. Treatment options of dumping syndrome include diet, medications, and surgical revision.
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10617
- synonyms: Dumping syndrome(NONE)
